spec/integration/filtering_spec.rb

```diff
@@ -6,8 +6,9 @@
 
   it 'prints a rerun command for shared examples in external files that works to rerun' do
     write_file "spec/support/shared_examples.rb", """
-      RSpec.shared_examples 'a failing example' do
-        example { expect(1).to eq(2) }
+      RSpec.shared_examples 'with a failing example' do
+        example { expect(1).to eq(2) } # failing
+        example { expect(2).to eq(2) } # passing
       end
```

Comment:
That's funny...I was wondering what the triple quotes were, honestly.  Someone else started that pattern for these integration specs and I just followed it w/o taking the time to look into the triple quotes.  I suspect the person who started that here brought it over from our cukes, where it's certainly valid.

We should probably drop the triple quotes at some point.  Separate PR, though.